File: Chapter03/ArmoredCar.lua
-- function ArmoredCar
function ArmoredCar()
	-- configure motors B and C to run in "brake" mode
	nxt.OutputSetMode(2,2)
	nxt.OutputSetMode(3,2)
	-- loop forever
	while(1) do
		-- set the power of the motors to 75 %
		-- 32 means "run mode"
		nxt.outputSetSpeed(2, 32, 75, 0)
		nxt.outputSetSpeed(3, 32, -75, 0)
	end
end

-- run the ArmoredCar function
ArmoredCar()

File: Chapter03/ArmoredCar.java
package org.nxtdavinci.armoredcar;

import lejos.nxt.*;

public class ArmoredCar {
	public static void main(String[] args) {
		// switch on speed regulation
		Motor.B.regulateSpeed(true);
		Motor.C.regulateSpeed(true);
		// set the speed of the motors to 600 rpm
		Motor.B.setSpeed(600);
		Motor.C.setSpeed(600);
		// loop forever
		while(true) {
			// run the motors in opposite direction
			Motor.B.forward();
			Motor.C.backward();
		}
	}
}